doc/README.yaml.md

```diff
@@ -0,0 +1,240 @@
+of its dependencies, each of which is found in a separate "feedstock" repository
+within the broader open-ce project.
+Default YAML files are provided for each of several environments in the open-ce
+repository, within https://github.com/open-ce/open-ce/tree/master/envs
+These files are named for their package environments, e.g.
+[tensorflow-env.yaml](https://github.com/open-ce/open-ce/blob/master/envs/tensorflow-env.yaml)
+or [pytorch-env.yaml](https://github.com/open-ce/open-ce/blob/master/envs/pytorch-env.yaml),
+among others. You can review these files as examples.
+
+Each environment file will contain a "packages" stanza which will list a number
+of feedstock repositories that are needed to build as requisites. Each feedstock
+dependency is listed on a separate line in the stanza. For example, in
+[xgboost-env.yaml](https://github.com/open-ce/open-ce/blob/master/envs/xgboost-env.yaml):
+```
+packages:
+  - feedstock : nccl              #[build_type == 'cuda']
+  - feedstock : xgboost
+```
+Here you see two feedstocks listed; these correspond to the open-ce repository
+tree for [nccl-feedstock](https://github.com/open-ce/nccl-feedstock) and
+[xgboost-feedstock](https://github.com/open-ce/xgboost-feedstock) respectively.
+Here, notice the `cuda` designation on the nccl feedstock line. This will only
+be built if the `build_type` is set to `cuda` when executing the `build_env.py`
+script, so it is considered an optional dependency that you may want to include
+if your runtime environment has CUDA available.
+
+Another stanza type that you might see in some environment YAML files is
+`imported_envs`. An example found in
+[horovod-env.yaml](https://github.com/open-ce/open-ce/blob/master/envs/horovod-env.yaml)
+looks like:
+```
+imported_envs:
+  - pytorch-env.yaml
+  - tensorflow-env.yaml
+```
+This is straightforward; it simply imports the contents of the listed yaml
+files. So if you are building the horovod environment, you will get a build of
+both pytorch and tensorflow as well because of the specified imported
+environments listed in this section.
+
+Another optional feature that you might use in some customization scenarios is the
+`channels` specifier.  If you are creating an environment file and you have a need
+to add a package dependency that is not found within open-ce but which is available
+at some other external delivery channel, you can specify the URL to the desired
+channel as seen below, placing this at the top of the file (and replacing this
+sample URL with the one you want to use):
+```
+channels:
+  - https://public.dhe.ibm.com/ibmdl/export/pub/software/server/ibm-ai/conda/
+```
+
+These stanza types are normally the only YAML elements that are used for the open-ce
+`build_env.py` builds. However, as each feedstock in the list is built, other YAML
+file information is used from the yaml files of each respective feedstock repository.
+
+
+# build_feedstock YAML file (meta.yaml)
```

Comment:
The following section should be in a separate file. I'm not positive if we need it, or if we should just point to the conda documentation. But it might be nice to have still.

Reply:
What we need documentation for is the build-config.yaml file that may appear in some feedstocks.

Reply:
OK - this is definitely the biggest change, because it basically throws away the bottom 2/3 of the file, if we simply don't want to document this here.

Will anyone want this anyway, even if in a separate file?  We can leave it out (less to maintain) and nobody might care -- and if we decided to add it, the content should be retrievable from the file change history.

The one change *NOT* done here yet is the last part requesting documentation for feedstock build-config.yaml files, which is new content, so I need to write something up first.